guide/src/python_from_rust.md

```diff
@@ -236,6 +236,75 @@ def leaky_relu(x, slope=0.01):
 # }
 ```
 
+### Want to embed Python in Rust with exposed modules?
+
+Internally Python maintains the `sys.modules` dict.
+An import in Python is simply a lookup into this dict.
```

Comment:
That is a bit misleading as stated - if the lookup fails, Python's import machinery takes over and tries to locate the module on the filesystem - going through `sys.path`, trying import hooks etc. (All of which doesn't find modules simply created by `PyModule_New`, which is the problem here.)

Reply:
This is true, but probably not relevant here?

I don't think it's misleading but I'm happy to hear what others think.

Reply:
OK, let me rephrase my comment in a less polite manner: calling all imports "simply a lookup" completely misrepresents what an import is. This is a wrong statement and should not go in like this.

Reply:
I take your point. That said, mentioning scanning the file system and running hooks seems to me to not be the purpose of this particular doc.

How about something like:

The python interpreter starts with a number of built-in modules being available in `sys.dict`. It is necessary to insert a newly created PyModule into this dictionary.